media/js/mozorg/home/home.js

```diff
@@ -40,15 +51,16 @@
     }
 
     function playVideo(e) {
-        var $card = $(this);
-        var $content = $card.next().find('.card-video-content');
-        var title = $card.find('.card-title').text();
+        var card = e.currentTarget;
+        var title = card.querySelector('.mzp-c-card-title').innerText;
+        var sibling = getNextEl(e.currentTarget);
+        var content = sibling.querySelector('.mzp-c-card-video-content');
 
-        if ($content.length) {
+        if (content) {
             e.preventDefault();
-            var video = $content.find('video')[0];
+            var video = content.querySelector('video');
 
-            Mozilla.Modal.createModal(this, $content, {
+            Mozilla.Modal.createModal(this, content, {
```

Comment:
This home page iteration doesn't have any video modals, but I'm going to leave this code here as I expect content to change fairly frequently.